pkg/datastore/influx.go

```diff
@@ -57,5 +58,9 @@ func (i *Influx) ExportTo(tmpdir string) (string, error) {
 		return "", err
 	}
 
+	for _, f := range backupFiles {
+		os.Remove(f)
```

Comment:
Never mind :D According to the pkg `os.Remove` only returns an `os.PathError`. I think we can assume that the path is correct :D